lib/vehiclelinkApi.js

```diff
@@ -6,41 +6,45 @@ class VehiclelinkApi {
     this.bearerToken = bearerToken;
   }
 
-  makeGetRequest() {
-    return {
+  makeGetRequest(opts) {
+    var request = {
```

Comment:
I think this should be replaced with let

Reply:
Fixed